src/conductor/client/http/models/sub_workflow_params.py

```diff
@@ -62,9 +62,6 @@ def name(self, name):
         :param name: The name of this SubWorkflowParams.  # noqa: E501
         :type: str
         """
-        if name is None:
```

Comment:
why are validations like this one unnecessary?

Reply:
Looks like there were a few changes at the server from when this code was generated. A test was failing due to missing parameters.

Error:
```
#13 127.9 2023-02-15 23:10:48,450 [1] conductor.client.http.api_client DEBUG    failed to deserialize, reason: Invalid value for `timeout_seconds`, must not be `None`
```

Reply:
I see
On one hand it would be pretty helpful to discover the issue early and fail at the client, on another hand, unless it's something customers struggle with, it doesn't matter, since server is still the source of the truth when it comes to validations

Reply:
Agreed, I think it might be a good to have when the server code gets more frequently generated, which means, it will be always up to date with the server.